Projects/Feature/HomeFeature/Sources/Scene/View/MealContentStackView.swift

```diff
@@ -20,9 +20,20 @@ final class MealContentStackView: UIStackView {
 
     public func updateContent(meals: [String]) {
         self.removeAllChildren()
-
-        guard !meals.isEmpty else { return }
-        let mealViews = meals.map { DotoriLabel($0) }
+        
+        var mealViews: [DotoriLabel] = []
+        
+        if meals.isEmpty {
+            mealViews.append(DotoriLabel("급식이 없습니다."))
+            mealViews.append(DotoriLabel(""))
+            mealViews.append(DotoriLabel(""))
+            mealViews.append(DotoriLabel(""))
+            mealViews.append(DotoriLabel(""))
+            mealViews.append(DotoriLabel(""))
+            mealViews.append(DotoriLabel(""))
```

Comment:
append 코드가 없으면 오류가 나나요? 왜 이렇게 코드를 작성한 건지 궁금해요!

Reply:
디자인에 "급식이 없습니다"가 View의 윗부분에 있어서 비어있는 DotoriLabel을 채워주면서 "급식이 없습니다"를 뷰의 윗부분에 보여주려고 저렇게 추가했어요!

Reply:
오호 그럼 ""를 저렇게 View여러개로 보내는 거 보다
```suggestion
            Array(repeating: DotoriLabel(""), count: 7)
```
이렇게 한 번 진행해보는건 어떨까요?!